.github/workflows/ci.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           npx @arethetypeswrong/cli --pack .
 
           # Run publint
-          npx publint . --strict
+          npx publint@0.2.12 . --strict
```

Comment:
@joshblack  wondering how you feel about pinning this version? looks like 0.3.0> is not compatible with yarn 1: https://github.com/publint/publint/releases/tag/publint%400.3.0

> If you use yarn 1, you should upgrade to the latest yarn version or a different package manager. Otherwise, no other changes are required for this new behavior.

Reply:
Sounds good to me! 👍

Reply:
Thanks both.

I don’t have merge permission in this repo so if you’re happy can you please merge to unblock https://github.com/primer/octicons/pull/1063?

(Feel free to also update and merge that PR 😅)